internal_transfers/actions/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Dependency directories
```

Comment:
What's the reasoning behind this in its own `.gitignore`?

Reply:
The scaffolding of the project template just came with this. It's really a project within a project and I don't know how to best structure it. Open for suggestions but I feel like giving it's own configuration keeps it independent and more Modular in case it moves later.

Reply:
Thanks for the explanation.

It seems like a case could be made for it being an independent project then.

Reply:
I just don't want to have to maintain yet another project.